nixos/modules/services/databases/lldap.nix

```diff
@@ -107,8 +107,17 @@ in
       wants = [ "network-online.target" ];
       after = [ "network-online.target" ];
       wantedBy = [ "multi-user.target" ];
+      script = lib.optionalString (!cfg.settings ? jwt_secret) ''
```

Comment:
You're checking two env vars below, but only one attr in `settings` above -- is that correct? (There is no *file option in `cfg.settings`?)

Reply:
Yes. From what I know `_FILE` is only available for environment variables and are invalid in the `configuration.toml`.